src/validation.cpp

```diff
@@ -748,7 +743,7 @@ static bool AcceptToMemoryPoolWorker(const CChainParams& chainparams, CTxMemPool
             const uint256 &hashAncestor = ancestorIt->GetTx().GetHash();
             if (setConflicts.count(hashAncestor))
             {
-                return state.DoS(10, false,
+                return state.DoS(10, ValidationInvalidReason::CONSENSUS, false,
```

Comment:
MISSING_INPUTS and CONFLICT imply that its possible the transaction may be valid, but we're missing something or otherwise unable to add it to the mempool right now. That is not the case here - there is no way this transaction could have been included in the mempool of any well-behaved node, nor could it ever make it into a block, so CONSENSUS is correct here.

Reply:
Ah, I see. The block "missing inputs" case is also reported as CONSENSUS; makes sense.